cleanup.php

```diff
@@ -1,51 +1,58 @@
 #!/usr/bin/env php
-
 <?php
+if (php_sapi_name() !== 'cli') {
+    http_response_code(403);
+    exit();
+}
 
-    require_once 'config.php';
-    require_once 'db_helper.php';
-    require_once 'osm_photo_note.php';
+require_once 'db_helper.php';
+require_once 'osm_photo_note.php';
 
-    $db_helper = new DBHelper();
+$db_helper = new DBHelper();
 
-    $old_inactive_photos = $db_helper->get_and_delete_old_inactive_photos();
-    foreach($old_inactive_photos as $photo) {
-        $file_name = $photo['file_id'] . $photo['file_ext'];
-        unlink($PHOTOS_TMP_DIR . '/' . $file_name);
+function deleteFile($file_path)
+{
+    echo "Deleting $file_path\n";
```

Comment:
stray echo?

Reply:
Also, I think it is completely reasonable that a warning is emitted when the file does not exist, why this check?

Reply:
Console output for logging or debugging, optional.

Reply:
```php
function deleteFile($file_path)
{
    if (file_exists($file_path)) {
        echo "Deleting $file_path\n";
        unlink($file_path);
    } else {
        echo "$file_path not found\n";
    }
}
```

Reply:
I mean, you replaced a warning when a file could not be deleted (emitted by `unlink`) with always logging when a file is deleted but not warning if the deletion failed.

Reply:
Why not 
```
unlink($file_path);
```
then?

Reply:
Normally the error reporting for PHP is disabled in production, so `unlink` would fail silently.